src/Umbraco.Web.UI.Client/src/controllers/navigation.controller.js

```diff
@@ -231,11 +247,11 @@ function NavigationController($scope, $rootScope, $location, $log, $q, $routePar
     }));
 
     //when the application is ready and the user is authorized setup the data
-    evts.push(eventsService.on("app.ready", function(evt, data) {
-        $scope.authenticated = true;
-        
-        // load languages
-        languageResource.getAll().then(function(languages) {
+    evts.push(eventsService.on("app.ready", function (evt, data) {
```

Comment:
@RobertCopilau why is this function empty that is listening for the `app.ready` event to be emitted?

That contained the loadLanguages & the bool for  `$scope.authenticated = true;` that has been moved out & moved further up?!

Reply:
Ah, the loadLanguages function used to be there before I moved it into "appState.sectionState.changed" listener.